dictionary.md

```diff
@@ -270,32 +270,59 @@ WARNING! Options in this section may change without notice, and should never be
 
 ## Deconvolution
 
+**beam_max_threshold**: Completely mask all pixels below this beam threshold during deconvolution. <br />
+  -*Default*: 1E-4 <br />
+  
+**beam_threshold**: Fraction of beam max value below which to mask the beam for deconvolution. 0.05 is really as far down as you should go with our current MWA beam models. This keyword is also used for calibration and modeling, but will be overwritten by `beam_model_threshold` or `beam_cal_threshold` if provided, and will be overwritten to 0.01 for those steps if `allow_sidelobe_sources` is turned on. This keyword is also used for beam modeling in some simulation code. <br />
+  -*Default*: 0.05
+
+**check_iter** Sets how frequently deconvolution will do a convergence check. <br />
+  -*Default*: 5 (check every 5th iteration)
+
 **deconvolve**: run fast holgraphic deconvolution. <br />
   -*Turn off/on*: 0/1 <br />
   -*Default*: 0 !Q <br />
   -*eor_wrapper_defaults*: 0 <br />
 
+**deconvolution_add_threshold**: Used in source detection to also fit additional components brighter than this threshold, written as percentage of max pixel flux. <br />
```

Comment:
I don't understand this one. Does this mean that those bright additional components will be treated as their own source?

Reply:
I'm not 100% sure - it's used in `fhd_source_detect`  and I based my description mostly on comments in the code, but based on a quick scan I _think_ it's basically saying for each iteration it finds the brightest component, and then looks for additional components nearby that are at least 80% as bright as the first component. I would need to read the code much more carefully to be confident in that answer though